point-of-sale-app/kubernetes-manifests/inventory-configmap.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: inventory-configmap
+data:
+  CONNECTOR: "IN_MEMORY"
+  ACTIVE_ITEM_TYPE: "mcdonalds"
```

Comment:
I recommend replacing the two well-known brand names - as well as the well known product names - with mock company/product names. Also why does the inventory span two different brand names?

Reply:
Removed references in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/2/commits/945912a1e0b704603502a37fb783e622f62d6454. The idea is to be able to switch between different store-types/enterprise types without having to restart or do any major changes. These two types are here as examples

Reply:
is that switching use case something that was specifically requested? I'm trying to think of why this would be needed. wouldn't a single physical store (ie. one instance of inventory service) correspond to one store? not two?

Reply:
Yes it was requested by @mike-ensor, as an easy way to use same deployment between multiple demos without having to restart the application. 

You are correct! This feature is not reflective of what will be out there in real; but a hidden API to enable quick switching between different kinds of store types using the same deployment without re-deploying multiple times. It's solely for convenience purpose. The API is not exposed in the UI. The idea is to have one system on multiple ABM clusters on emulated baremetal in GCE and continue to use it for multiple demos with different store types.